api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -583,6 +584,10 @@ public void initializeInMemoryDatabase() throws SQLException {
 			throw new RuntimeException(
 			        "You shouldn't be initializing a NON in-memory database. Consider unoverriding useInMemoryDatabase");
 
+		//Because creator property in the superclass is mapped as nullable by the autoddl tool, we need to first
```

Comment:
I need to rephrase this, meant to say:

Because creator property in the superclass is mapped with optional set to false, the autoddl tool marks the column as not nullable